source/css/scss/app/pages/monthly-good.scss

```diff
@@ -97,6 +97,77 @@
 			}
 		}
 	}
+	//FAQ style needs to be added. Commented out to prevent merge conflict
+	//.faq-container {
+	//	margin-top: 25px;
+	//}
+
+	.more-about-kiva {
+		.more-about-kiva-headline {
+			color: $kiva-green;
+			text-align: center;
+
+			@media #{$large-up} {
+				text-align: left;
+			}
+		}
+
+		.more-about-kiva-text {
+			@media #{$large-up} {
+				float: left;
+			}
+		}
+
+		.img-container {
+			@media #{$large-up} {
+				float: right;
+			}
+
+			.charity-nav-wrapper {
+				float: left;
+
+				@media #{$large-up} {
+					float: none;
+				}
+
+				p:first-child {
+					margin-bottom: 0;
+				}
+
+				img {
+					width: rem-calc(100);
+
+					@media #{$large-up} {
+						width: rem-calc(100);
```

Comment:
Is this media query still needed? Was it supposed to be a different value?

Reply:
Thanks for catching that, got a little carried away with the media queries.